aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -254,15 +254,15 @@ struct TORCH_CUDA_CPP_API RNNDescriptor : public Descriptor<
                                              &cudnnCreateRNNDescriptor,
                                              &cudnnDestroyRNNDescriptor> {
   DropoutDescriptor dropout_desc_;
-  void set(cudnnHandle_t handle, int hidden_size, int proj_size, int num_layers, DropoutDescriptor&& dropout_desc,
+  void set(cudnnHandle_t handle, c10::SymInt hidden_size, c10::SymInt proj_size, int num_layers, DropoutDescriptor&& dropout_desc,
```

Comment:
These changes shouldn't have been necessary, the Descriptor classes should only get called from CUDA, which can take plain integers.